ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/CVEsTable.tsx

```diff
@@ -1,48 +1,230 @@
+                        distroTuples,
+                        topCVSS,
+                        affectedNodeCount,
+                        firstDiscoveredInSystem,
+                    } = nodeCve;
+                    const isExpanded = expandedRowSet.has(cve);
+
+                    const prioritizedDistros = sortCveDistroList(distroTuples);
+                    const summary =
+                        prioritizedDistros.length > 0 ? prioritizedDistros[0].summary : '';
+                    const scoreVersions = getScoreVersionsForTopCVSS(topCVSS, distroTuples);
+
+                    return (
+                        <Tbody key={cve} isExpanded={isExpanded}>
+                            <Tr>
+                                <Td
+                                    expand={{
+                                        rowIndex,
+                                        isExpanded,
+                                        onToggle: () => expandedRowSet.toggle(cve),
+                                    }}
+                                />
+                                <Td dataLabel="CVE" modifier="nowrap">
+                                    <Link to={getNodeEntityPagePath('CVE', cve)}>{cve}</Link>
+                                </Td>
+                                <Td dataLabel="Nodes by severity">
+                                    <SeverityCountLabels
+                                        criticalCount={critical.total}
+                                        importantCount={important.total}
+                                        moderateCount={moderate.total}
+                                        lowCount={low.total}
+                                        // TODO - Add filtered severities once filter toolbar is in place
+                                    />
+                                </Td>
+                                <Td dataLabel="Top CVSS">
+                                    <CvssFormatted
+                                        cvss={topCVSS}
+                                        scoreVersion={
+                                            scoreVersions.length > 0
+                                                ? scoreVersions.join('/')
+                                                : undefined
+                                        }
+                                    />
+                                </Td>
+                                <Td dataLabel="Affected nodes">
+                                    {affectedNodeCount} / {totalNodeCount} affected nodes
+                                </Td>
+                                <Td dataLabel="First discovered">
+                                    <DateDistance date={firstDiscoveredInSystem} />
+                                </Td>
+                            </Tr>
+                            <Tr isExpanded={isExpanded}>
+                                <Td />
+                                <Td colSpan={colSpan - 1}>
```

Comment:
Curious as to why we need to subtract the magic number `1` here?

Reply:
The `- 1` is to account for the empty `<Td />` above it, which is used to maintain alignment of the text with the leftmost data column (excluding the collapse button column).
![image](https://github.com/stackrox/stackrox/assets/1292638/982f2f8a-6889-4c1b-8987-83e8b3c6f69b)

This doesn't cause any issues with the aXe devtools, but I do notice a good handful of findings that I think are new since the PF 5 upgrade.